typing/typedecl.ml

```diff
@@ -2881,27 +2489,11 @@ let transl_with_constraint id ?fixed_row_path ~sig_env ~sig_decl ~outer_env
 
 (* Approximate a type declaration: just make all types abstract *)
 
-<<<<<<< HEAD
 let abstract_type_decl ~injective jkind params =
   let arity = List.length params in
-  Ctype.begin_def();
   let params = List.map Ctype.newvar params in
```

Comment:
This `newvar` looks like it has moved from within the `begin_def` to outside of it.

Reply:
well spotted